examples/bq-exfil-demo/versions.tf

```
@@ -18,7 +18,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 3.0"
+      version = "< 5.0"
```

We can keep this as since it has the pinned module

```suggestion
      version = "~> 3.0"
```